### DISCUSSION TITLE
I can't update data (column does not exist)

### DISCUSSION THREAD

**Discussion by Bigmancozmo** [supabase/supabase#12750] [Questions] [answered]
I'm making a Discord bot, and I'm using Supabase for the databases, I've been trying to fix this for a few hours now, and I can't get it to work.
```js
async function setServerData(serverId, sData) {
    const {
        data,
        error
    } = await supabase
        .from('server_data')
        .update(sData.xp)
        .eq('id', toString(serverId))
    console.log(error)
}
```
And my error is 
```
{
  code: '42703',
  details: null,
  hint: null,
  message: 'column _.837463896800624702 does not exist'
}
```

**Comment by GaryAustin1** [comment, marked as answer]
I moved this to discussions.  You really might be better served asking in Supabase Discord.
This is highly unlikely a bug based on your column being named _.837.....   It is likely the value you are passing is not an object with col:value for the update to use.  Of course you might have a column named that.  Does console.log(sData.xp) show an object with column name and value?

**Reply by Bigmancozmo** [reply]
Yes, this helped me find my issue. I had mixed up rows & columns.
